src/components/cart/CartItem/CartItem.tsx

```diff
@@ -20,30 +21,31 @@ function CartItem({ item }: Props) {
   const { updateItemQuantity } = useCart()
 
   return (
-    <article
+    <Card
       className="cart-item"
-      data-cart-item
       data-testid="cart-item"
       data-sku={item.itemOffered.sku}
       data-seller={item.seller.identifier}
     >
-      <section data-cart-item-content>
-        <Image
-          baseUrl={item.itemOffered.image[0].url}
-          alt={item.itemOffered.image[0].alternateName}
-          sourceWidth={360}
-          aspectRatio={1}
-          width={72}
-          breakpoints={[50, 100, 150]}
-          layout="constrained"
-          backgroundColor="#f0f0f0"
-          options={{
-            fitIn: true,
-          }}
-        />
+      <CardContent>
+        <CardImage>
+          <Image
+            baseUrl={item.itemOffered.image[0].url}
+            alt={item.itemOffered.image[0].alternateName}
+            sourceWidth={360}
+            aspectRatio={1}
+            width={72}
+            breakpoints={[50, 100, 150]}
+            layout="constrained"
+            backgroundColor="#f0f0f0"
+            options={{
+              fitIn: true,
+            }}
+          />
+        </CardImage>
         <div data-cart-item-summary>
           <p className="text-body">{item.isVariantOf?.name}</p>
-          <div data-cart-item-price>
+          <span data-cart-item-price>
```

Comment:
```suggestion
          <span data-cart-item-prices>
```

Just because it wrappers more than one price 👀

Reply:
If you agree with this suggestion, don't forget to replace it on the css too, ok? 😉